src/main/kotlin/lotto/LottoNumbersGenerator.kt

```diff
@@ -0,0 +1,5 @@
+package lotto
+
+class LottoNumbersGenerator(var strategy: LottoNumbersGenerateStrategy) {
+    fun generate(): LottoNumbers = strategy.generate()
+}
```

Comment:
제가 제안 드린 방식은, 이 Ganerator가 수동 티켓 발급까지 갖고 있게 만드는 것이었어요!
아래 처럼 생성기자체는 한 곳에서 몰아서 수행하고,
숫자를 랜덤으로 생성하는 전략만 따로 받아서, 이 전략으로 자동 티켓을 발급하게 만들어보는 것을 의도했어요.

```suggestion
class LottoNumbersGenerator(val strategy: LottoNumbersGenerateStrategy) {
    fun generateAutoTicket(): LottoNumbers = strategy.generate()

    fun generateManualTicket(numbers: List<Int>): LottoNumbers = ...
}

저도 수민님 처럼 만들어본 적도 있는데, 수동 티켓을 만드는 객체를 한 티켓 당 하나씩 만들어야하는 비효율이 존재하더라구요.
수동 티켓을 만드는 데 하나의 객체만 필요하다고 판단해서 저는 위와 같이 작성하게 되었습니다 :)